pajbot/modules/basic/masspoints.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from pajbot.managers.db import DBManager
+from pajbot.models.command import Command
+from pajbot.models.command import CommandExample
+from pajbot.models.user import User
+from pajbot.modules import BaseModule
+from pajbot.modules import ModuleSetting
+from pajbot.modules.basic import BasicCommandsModule
+
+from sqlalchemy import text
+
+log = logging.getLogger(__name__)
+
+
+class MassPointsModule(BaseModule):
+    ID = __name__.split(".")[-1]
+    NAME = "Mass Points"
+    DESCRIPTION = "Give points to everyone watching the stream"
+    CATEGORY = "Feature"
+    SETTINGS = [
+        ModuleSetting(
+            key="sub_points",
+            label="Points to give for subscribers for each pleb point",
+            type="number",
+            required=True,
+            placeholder="1",
+            default=1,
+            constraints={"min_value": 0, "max_value": 100},
+        )
+    ]
+
+    def load_commands(self, **options):
+        self.commands["masspoints"] = Command.raw_command(
```

Comment:
I don't necessarily agree with the command being `!masspoints`, it should be something more descript